### DISCUSSION TITLE
In regards to art assets, why not use AI?

### DISCUSSION THREAD

**Discussion by dorkster** [flareteam/flare-game#941] [General]
I was asked this question today, and I think it's a fair question. I just wanted to share my response publicly and see what the community here thinks.

> The short answer is "no". Here's the long answer:
> 
> (In no particular order of importance)
> 
> 1. Incompatible with Creative Commons "remix culture". We provide the source files (blend files, textures, etc) for most if not all of our assets. AI only gives us the end result to distribute.
> 2. Not reasonable to use for animations. Every AI animation I've seen has large inconsistencies between frames. But let's ignore this for now and assume we only want to target static 2D images.
> 3. If we don't make changes to the player model, I don't think it would be appropriate to make variations of the equipment icons. As of now, they (for the most part) accurately represent the item as it will be presented on the player model.
> 4. We don't want to invoke potential copyright issues. Is the AI trained on copyrighted material? What's the copyright on the output? If we insert our assets, will the AI continue to use and remix the data for other users, stripping any attribution and thus violating the CC-BY-SA license? I guess several of these could be solved by using a self-hosted solution (Stable Diffusion? I'm not up-to-date with AI stuff)
> 5. Not enough assets to train our own model. If we had enough to do this, I don't think we'd need AI in the first place. In other words: AI would be great if we had 100 armors and wanted 1,000. But not so much if we have 10 and want 100.

**Comment by clintbellanger** [comment]
Agreed
